src/Service.Tests/GraphQLBuilder/QueryBuilderTests.cs

```diff
@@ -387,6 +387,7 @@ string expectedNameInDescription
         [DataRow(GraphQLOperation.Mutation, new[] { Config.Operation.Execute }, new[] { "execute" }, false, DisplayName = "Query field not generated because the configured operation is mutation.")]
         public void StoredProcedureEntityAsQueryField(GraphQLOperation? graphQLOperation, Config.Operation[] operations, string[] permissionOperations, bool expectsQueryField)
         {
+            string entityName = "MyStoredProcedure";
             string gql =
             @"
             type StoredProcedureType @model(name:""MyStoredProcedure"") {
```

Comment:
entityName variable can be used here as well.

Reply:
updated.